live/src/agama-installer.kiwi

```diff
@@ -108,7 +108,7 @@
         <package name="systemd"/>
         <package name="procps"/>
         <package name="iputils"/>
-        <package name="hyper-v"/>
+        <package name="hyper-v" arch="x86_64"/>
```

Comment:
Hyper-v is available on aarch64, should we include it as well?

Reply:
Looks reasonable.